src/Umbraco.Tests/FrontEnd/UmbracoHelperTests.cs

```diff
@@ -1,83 +1,90 @@
 
             Assert.AreEqual("Hello world, this is some&hellip;", result);
         }
 
-        /// <summary>
-        /// If a truncated string ends with a space, we should trim the space before appending the ellipsis.
-        /// </summary>
         [Test]
-        public void Truncate_Simple_With_Trimming()
+        public static void When_truncating_a_string_ends_with_a_space_we_should_trim_the_space_before_appending_the_ellipsis()
         {
-            var text = "Hello world, this is some text <a href='blah'>with a link</a>";
-
             var helper = new UmbracoHelper();
 
-            var result = helper.Truncate(text, 26).ToString();
+            var result = helper.Truncate(SAMPLE, 26).ToString();
 
             Assert.AreEqual("Hello world, this is some&hellip;", result);
         }
 
         [Test]
-        public void Truncate_Inside_Word()
+        public static void Truncate_Inside_Word()
         {
-            var text = "Hello world, this is some text <a href='blah'>with a link</a>";
-
             var helper = new UmbracoHelper();
 
-            var result = helper.Truncate(text, 24).ToString();
+            var result = helper.Truncate(SAMPLE, 24).ToString();
 
             Assert.AreEqual("Hello world, this is som&hellip;", result);
         }
 
         [Test]
-        public void Truncate_With_Tag()
+        public static void Truncate_With_Tag()
         {
-            var text = "Hello world, this is some text <a href='blah'>with a link</a>";
-
             var helper = new UmbracoHelper();
 
-            var result = helper.Truncate(text, 35).ToString();
+            var result = helper.Truncate(SAMPLE, 35).ToString();
 
             Assert.AreEqual("Hello world, this is some text <a href='blah'>with&hellip;</a>", result);
         }
 
         [Test]
-        public void Create_Encrypted_RouteString_From_Anonymous_Object()
+        public static void Create_Encrypted_RouteString_From_Anonymous_Object()
         {
             var additionalRouteValues = new
             {
                 key1 = "value1",
                 key2 = "value2",
                 Key3 = "Value3",
                 keY4 = "valuE4"
-        };
-            var encryptedRouteString = UmbracoHelper.CreateEncryptedRouteString("FormController", "FormAction", "", additionalRouteValues);
+            };
+
+            var encryptedRouteString = UmbracoHelper.CreateEncryptedRouteString(
+                "FormController",
+                "FormAction",
+                "",
+                additionalRouteValues
+                );
+
             var result = encryptedRouteString.DecryptWithMachineKey();
-            var expectedResult = "c=FormController&a=FormAction&ar=&key1=value1&key2=value2&Key3=Value3&keY4=valuE4";
 
-            Assert.AreEqual(expectedResult, result);
+            const string EXPECTED_RESULT = "c=FormController&a=FormAction&ar=&key1=value1&key2=value2&Key3=Value3&keY4=valuE4";
```

Comment:
Variable name convention for constants are the same as for normal vars, so we'd use `expectedResult`.